test-requirements.txt

```diff
@@ -0,0 +1 @@
+odoo14-addon-base_product_mass_addition @ git+https://github.com/OCA/product-attribute@refs/pull/864/head#subdirectory=setup/base_product_mass_addition
```

Comment:
remove this when PR is merged : https://github.com/OCA/product-attribute/pull/864